### PR TITLE
Adds function to redirect when / at end

### PR DESCRIPTION
**Pull request by NickSNFK**

### CODE REVIEW

File: socialDistribution/app/static/app/js/profile.js [hunk 1]
```diff
@@ -135,31 +135,31 @@ function sendFollowRequestToInbox(e){
...
             const addToSentRequestURL = new URL("add-to-sent", `${window.location.protocol}//` + window.location.host);
-            
+
             const sentRequestObject = {user_id:user_id, author_id:author_id};
 
             fetch(addToSentRequestURL, {
                 method: "POST",
                 headers: new Headers({
-                'Authorization': 'Basic '+btoa('server1:123'), 
+                'Authorization': 'Basic '+btoa('server1:123'),
```
Comment (saadman13) [inline socialDistribution/app/static/app/js/profile.js:162]: These will fail now. as we don't have server1 anymore as a username:password.

File: socialDistribution/app/static/app/js/profile.js [hunk 2]
```diff
@@ -183,7 +183,7 @@ async function getSingleAuthorInfo(url, token){
 
     if (token){
         headers = new Headers({
-                'Authorization': 'Basic '+token, 
+                'Authorization': 'Basic '+token,
```
Comment (saadman13) [inline socialDistribution/app/static/app/js/profile.js:186]: Are these meant to be part of these change?

**Review by saadman13 — APPROVED** [review]
The middleware changes looks good, so I have approved this. There are some other changes which I am not sure if its mistakenly added as a part of this (So I leave that to you)

File: socialDistribution/app/static/app/js/profile.js (hunk 2 above)
Comment (saadman13) [reply socialDistribution/app/static/app/js/profile.js:186]: Oh scratch this, looks to be just a space change

Comment (NickSNFK) [reply socialDistribution/app/static/app/js/profile.js:186]: my vs code likes to remove spaces lol

File: socialDistribution/app/static/app/js/profile.js (hunk 1 above)
Comment (NickSNFK) [reply socialDistribution/app/static/app/js/profile.js:162]: I think we do have this but this change was only for the space at the end from vs code.

**Pull request merged**